checked examples and added to working directory

BLSowmya opened the pull request:
<!--
👋 Hi, thanks for sending a PR to Scarpe! 💖
-->

### Description

I checked examples from not_checked directory and those examples which are working added to working directory and
<!-- Please write a summary of the change, Please also include relevant motivation and context: -->

### Image(if needed, helps for a faster review)

<!-- Add image of the feature u added here -->

### Checklist

- [x] Run tests locally

gintama91 reviewed (commented):
check this examples once: 

```
definr.rb (definr.com dont work anymore..?)
funnies.rb ( we have  a equivalent one working and here i think the issue is with xml..?)
image-icon.rb ( legacy url is no langer valid so maybe we have to change it? or leave it?)
list_box-select-class.rb (para text not updating)
```

apart from this remaining examples are working good.

BLSowmya commented:
> check these examples once:
> 
> ```
> definr.rb (definr.com dont work anymore..?)
> funnies.rb ( we have  a equivalent one working and here i think the issue is with xml..?)
> image-icon.rb ( legacy url is no langer valid so maybe we have to change it? or leave it?)
> list_box-select-class.rb (para text not updating)
> ```
> 
> apart from this remaining examples are working good.

I have changed the files and added new shoes logo to image-icon.rb file
